SnipsPlatform/SnipsPlatform.swift

```diff
@@ -71,25 +75,30 @@ public class SnipsPlatform {
     ///   - enableHtml: This will add html tags to `snipsWatchHandler`. `false` by default.
     ///   - enableLogs: Print Snips internal logs. This should be used only in Debug configuration. `false` by default.
     ///   - enableInjection: Enable ASR injection feature. You can add new entities using the `requestInjection` method. `false` by default.
+    ///   - enableASRPartialText: Enable ASR partial text capture feature. This is quite resource intensive and will affect the capacity of the platform to run in real time. `false` by default
     ///   - userURL: The platform will use this path to store data. For instance when using the ASR injection, new models will be stored in this path. By default it creates a `snips` folder in the user's document folder.
     ///   - g2pResources: When enabling injection, g2p resources are used to generated new word pronunciation. You either need g2p data or a lexicon when injecting new entities.
     ///   - asrModelParameters: Override default ASR model parameters
+    ///   - asrPartialTextPeriodMs: ASR partial text capture period in ms. `250ms` by default. You need to have ASR partial text capture enabled.
     /// - Throws: A `SnipsPlatformError` if something went wrong while parsing the given the assistant.
     public init(assistantURL: URL,
                 hotwordSensitivity: Float = 0.5,
                 enableHtml: Bool = false,
                 enableLogs: Bool = false,
                 enableInjection: Bool = false,
+                enableASRPartialText: Bool = false,
```

Comment:
I would rename this to `enableAsrPartialText` because we never kept acronyms in upper case so far (`enableHtml` for instance).
I don't necessarily disagree with the uppercase format but let's just be consistent on this and stay on the project code style.